lib/cli/driver.js

```diff
@@ -262,12 +273,130 @@ class DriverCommand {
     return this.config.installedDrivers;
   }
 
-  update (/*{driver}*/) {
-    // TODO implementation
-    return {};
+  /**
+   * Attempt to update one or more drivers using NPM
+   *
+   * @param [object] updateSpec - object containing a 'driver' key whose value is the name of
```

Comment:
We usually use the [typedef](https://github.com/appium/appium-adb/blob/master/lib/tools/adb-commands.js#L965) format to describe such things